IRremoteESP8266.cpp

```diff
@@ -108,10 +108,9 @@ void IRsend::begin() {
 }
 
 // Generic method for sending data that is common to most protocols.
-// Default to transmitting the Most Significant Bit (MSB) first.
 void IRsend::sendData(unsigned int onemark, unsigned long onespace,
                       unsigned int zeromark, unsigned long zerospace,
-                      uint32_t data, uint8_t nbits, bool MSBfirst = true) {
```

Comment:
I was planning to do that squash in the merge to master as a single commit.
I'll try to work out how to squash the commits prior to that. I'm not so used to git. My background has been Perforce. :-/